dspace-server-webapp/src/test/java/org/dspace/app/rest/CollectionGroupRestControllerIT.java

```diff
@@ -955,7 +956,10 @@ public void getCollectionItemReadGroupTest() throws Exception {
                             jsonPath("$", GroupMatcher.matchGroupEntry(role.getID(), role.getName())));
     }
 
+    // Put on ignore because there's no support to identify read rights on a group for a user in a special
+    // com/coll admin group
```

Comment:
I think we should create a ticket for this & link to the ticket here.  I'll create the ticket & add a link in this comment once it is ready.

Reply:
Thanks, if I remember correctly, @abollini recommended a single SQL query to solve this without a performance risk, it would be good if he can share his recommendation in this ticket (I don't think he identified the exact query yet)

Reply:
The ticket has now been referenced in each of the @Ignore statements